### ISSUE TITLE
How can we filter to show some specific apps? 

### ISSUE THREAD

**Issue by shwetarc** [nitish24p/react-native-upi#56]
Currently, using this library allows our App user to pay using all UPI Apps installed on his phone including WhatsApp pay, Banking Apps. Is it possible to show the user to pay through selected UPI Apps such as Paytm, Gpay only.

**Comment by nitish24p** [comment]
Yes you need to know package name of the concerned UPI apps while starting UPI intent, although you will have to fork this lib and make edits on the Java side

**Comment by Shweta-UMR** [comment]
@nitish24p Yes i have added some code in java side and its working now Thanks
